ci-operator/config/kubernetes/autoscaler/master.json

```diff
@@ -1,4 +1,14 @@
 {
+  "release_tools_config": {
```

Comment:
i'd prefer for this to be "cluster": ... like the others.  The image is already public.